docs/patterns/components/OptionsMenu/index.js

```diff
@@ -12,6 +12,11 @@ export default class Demo extends Component {
     return (
       <div>
         <h1>Options Menu</h1>
+        <h2>Component Description</h2>
```

Comment:
This is another doc page that we could make an issue for using `Demo` component.